TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/automation/AndroidAccessibilitySettingsRetriever.java

```diff
@@ -0,0 +1,42 @@
+package com.novoda.tpbot.automation;
+
+import android.content.ContentResolver;
+import android.provider.Settings;
+import android.util.Log;
+
+public class AndroidAccessibilitySettingsRetriever {
+
+    private static final String EMPTY_STRING = "";
+
+    private final ContentResolver contentResolver;
+
+    public AndroidAccessibilitySettingsRetriever(ContentResolver contentResolver) {
+        this.contentResolver = contentResolver;
+    }
+
+    public String retrieveEnabledAccessibilityServices() {
+        if (canRetrieveAccessibilitySettings()) {
+            return getEnabledAccessibilityServices();
```

Comment:
Do you have to specify a feedback type for your service, even if you pass `FEEDBACK_ALL_MASK` here?